ziti/constants/constants.go

```diff
@@ -58,62 +60,64 @@ const (
+	PkiSignerCertVarDescription                      = "Path to the controller's edge signer CA cert"
 	PkiSignerKeyVarName                              = "ZITI_PKI_SIGNER_KEY"
-	PkiSignerKeyVarDescription                       = "Path to the Ziti Signing Key"
+	PkiSignerKeyVarDescription                       = "Path to the controller's edge signer CA key"
 	CtrlEdgeIdentityEnrollmentDurationVarName        = "ZITI_EDGE_IDENTITY_ENROLLMENT_DURATION"
 	CtrlEdgeIdentityEnrollmentDurationVarDescription = "The identity enrollment duration in minutes"
 	CtrlEdgeRouterEnrollmentDurationVarName          = "ZITI_ROUTER_ENROLLMENT_DURATION"
 	CtrlEdgeRouterEnrollmentDurationVarDescription   = "The router enrollment duration in minutes"
 	CtrlPkiEdgeCertVarName                           = "ZITI_PKI_EDGE_CERT"
-	CtrlPkiEdgeCertVarDescription                    = "Path to Identity Cert for Ziti Edge Controller"
+	CtrlPkiEdgeCertVarDescription                    = "Path to the controller's web identity client certificate"
 	CtrlPkiEdgeServerCertVarName                     = "ZITI_PKI_EDGE_SERVER_CERT"
-	CtrlPkiEdgeServerCertVarDescription              = "Path to Identity Server Cert for Ziti Edge Controller"
+	CtrlPkiEdgeServerCertVarDescription              = "Path to the controller's web identity server certificate, including partial chain"
 	CtrlPkiEdgeKeyVarName                            = "ZITI_PKI_EDGE_KEY"
-	CtrlPkiEdgeKeyVarDescription                     = "Path to Identity Key for Ziti Edge Controller"
+	CtrlPkiEdgeKeyVarDescription                     = "Path to the controller's web identity private key"
 	CtrlPkiEdgeCAVarName                             = "ZITI_PKI_EDGE_CA"
-	CtrlPkiEdgeCAVarDescription                      = "Path to Identity CA for Ziti Edge Controller"
+	CtrlPkiEdgeCAVarDescription                      = "Path to the controller's web identity root CA cert"
 	PkiAltServerCertVarName                          = "ZITI_PKI_ALT_SERVER_CERT"
-	PkiAltServerCertVarDescription                   = "Alternative server certificate to use. Must be specified with ZITI_PKI_ALT_SERVER_KEY"
+	PkiAltServerCertVarDescription                   = "Path to the controller's root identity alternative server certificate. Requires ZITI_PKI_ALT_SERVER_KEY"
 	PkiAltServerKeyVarName                           = "ZITI_PKI_ALT_SERVER_KEY"
-	PkiAltServerKeyVarDescription                    = "Key to use with the alternative server cert. Must be specified with ZITI_PKI_ALT_SERVER_CERT"
+	PkiAltServerKeyVarDescription                    = "Path to the controller's root identity alternative private key. Requires ZITI_PKI_ALT_SERVER_CERT"
 	ZitiEdgeRouterNameVarName                        = "ZITI_ROUTER_NAME"
-	ZitiEdgeRouterNameVarDescription                 = "The Edge Router Name"
+	ZitiEdgeRouterNameVarDescription                 = "A filename prefix for the router's key and certs"
```

Comment:
the old value was correct. it's the name "in" ziti. please put it back. it's just happenstance that the name is used to namespace files like this

Reply:
With this change, I'm working toward filling a particular gap between how this var is documented in the output of `ziti create config environment` and the fact that it's ignored by `ziti create config router`, and replaced by a required flag `--routerName` ([link to dev doc example of this confusing experience](https://github.com/openziti/ziti/blob/main/doc/003-local-deploy.md#initialize-an-edge-router)).

The result of any changes here should be a description that answers the question "How is this variable used by Ziti's built-in config generator?" because only users of the built-in config generator will see this description, not quickstart users.

I recognize the quickstarts ([link to line](https://github.com/openziti/ziti/blob/linux-package-router/ziti/cmd/edge/quickstart.go#L139)) do co-opt the same variable to set the router's entity name in the Ziti mgmt API, but that use case should be documented as part of the quickstarts since this config generator has other applications that do not co-opt the variable.